.github/actions/run-docker-container/action.yml

```diff
@@ -0,0 +1,44 @@
+---
+name: 🏃
+description: >-
+  Run Docker container to
+  upload Python distribution packages to PyPI
+inputs:
+  user:
+    description: PyPI user
+    required: false
+  password:
+    description: Password for your PyPI user or an access token
+    required: false
+  repository-url:
+    description: The repository URL to use
+    required: false
+  packages-dir:
+    description: The target directory for distribution
+    required: false
+  verify-metadata:
+    description: Check metadata before uploading
+    required: false
+  skip-existing:
+    description: >-
+      Do not fail if a Python package distribution
+      exists in the target package index
+    required: false
+  verbose:
+    description: Show verbose output.
+    required: false
+  print-hash:
+    description: Show hash values of files to be uploaded
+    required: false
+runs:
+  using: docker
+  image: "{{image}}"
+  args:
+  - ${{ inputs.user }}
+  - ${{ inputs.password }}
+  - ${{ inputs.repository-url }}
+  - ${{ inputs.packages-dir }}
+  - ${{ inputs.verify-metadata }}
+  - ${{ inputs.skip-existing }}
+  - ${{ inputs.verbose }}
+  - ${{ inputs.print-hash }}
```

Comment:
These aren't actually used. Somebody contributed them, but I don't see them being needed. So perhaps we shouldn't keep them around anymore.

Reply:
Aren't the inputs used by the entrypoint script?

https://github.com/pypa/gh-action-pypi-publish/blob/699cd6103f50bf5c3b2f070c70712d109c168e6c/twine-upload.sh#L37-L41

Reply:
@br3ndonland they are being read through the environment variables. This bit passes them as CLI arguments additionally but said interface isn't actually used anywhere. I think somebody contributed it and I accepted that PR, but there was never really any need for it. I believe it only brings confusion, and removing it would make that confusion go away.

It's probably a good idea to drop them from the current default branch separately, though.